Welcome page (cont.)

ixley opened the pull request:
- Adds partner logos to welcome page
- Comments out share/donate buttons until functionality is added
- also: Removes `src/js/config.js` which inadvertently got added as a tracked file again a couple months ago

DaleMcGrew commented:
Thank you Jeff. As much of a pain as it is to have the config.js file checked into source, we unfortunately need it so our automated Travis tests run. Are you able to revert the removal of config.js file?

ixley commented:
@DaleMcGrew Ah.. I didn't realize (obviously) - thought it was an accidental check in. I've reverted and removed it from the .gitignore file (which was partly what threw me off).

DaleMcGrew commented:
Thank you Jeff! 👍
